main/zoe/api/zoe.md

```diff
@@ -200,14 +245,133 @@ either the `proposal `or `payments` are empty, indicate this by
 omitting that argument or passing `undefined`, instead of passing an
 empty record.
 
-The `proposal` has three parts: 
-- `want`: An object with keywords as keys and `amounts` as values.
-- `give`: An object with keywords as keys and `amounts` as values.
-- `exit`: Specifies the payout-liveness policy Zoe can guarantee for the offer.
+`exit`'s value should be an `exitRule`, an object with three possible keys for
+key:value pairs:
+- `waived`: The user can't cancel and relies entirely on the smart contract to promptly finish their offer. Takes a `null` value.
+- `onDemand`: The user can cancel on demand. Takes a `null` value.
+- `afterDeadline`: The offer is automatically cancelled after a deadline, as determined by its `timer` and `deadline` properties and
+  `deadline`'s value, which is denoted in number of seconds.
```

Comment:
```suggestion
- `waived:null`: The user can't cancel and relies entirely on the smart contract to promptly finish their offer. 
- `onDemand:null`: The user can cancel on demand.
- `afterDeadline`: The offer is automatically cancelled after a deadline, as determined by its `timer` and `deadline` properties. The timer is a timer, and the deadline is with respect to the timer. Some example timers use Unix epoch time, while others count block height.
```

Do we have a description of timers anywhere? There should have been one with the REPL/wallet discussion, but I'm not sure where that is anymore.

Reply:
Not really. The REPL docs fell through cracks while waiting for Dean to decide about Registry, and by the time that happened I was working on other things. Also, the REPL docs deal with timerService, and don't have any references to a "timer" object that I see off hand. There is a SwingSet internal doc about it: https://github.com/Agoric/agoric-sdk/blob/027d4b6c2fd26f2631245fe1d53205ec8ee44ea7/packages/SwingSet/docs/timer.md

I'll make an issue that a "timer" section should be added to the Distributed Programming Guide (which is really more of a "Here's non-standard JS stuff you should know about" Guide : -))